apps/web/src/components/Editor.tsx

```diff
@@ -70,6 +70,9 @@ export const extensions = [
       width: 4,
       class: "text-slate-400",
     },
+    codeBlock: {
```

Comment:
Om detta funkar kan du ta bort `CodeBlockLowLight` extensionen

Reply:
I think we still need it for highlighting, right? It's just that the codeblock itself is included in the starterpack